DataProcessing/datax-host/src/main/scala/datax/sink/BlobSinker.scala

```diff
@@ -213,5 +229,37 @@ object BlobSinker extends SinkOperatorFactory {
     )
   }
 
+  private def getStorageAccountName(path:String):String ={
+    val regex = "@([a-zA-Z0-9-_]+).blob.core.windows.net".r
```

Comment:
.r? indent

Reply:
.r is to create regex. Created a constant in datax-core package and replaced every instance of (blob.core.windows.net) with the constant